src/pip/_internal/utils/misc.py

```diff
@@ -545,10 +545,7 @@ def __init__(self, lines):
 
     def readline(self):
         try:
-            try:
-                return next(self._gen)
-            except NameError:
-                return self._gen.next()
+            return next(self._gen)
```

Comment:
Does this work on py2?

Reply:
Yep, as also outlined in https://github.com/pypa/pip/pull/8405#discussion_r436247051

Reply:
Cool. 😬

Reply:
Cool. 😬